analysis/src/HistoManager.cxx

```diff
@@ -4,13 +4,16 @@
 #include "TClass.h"
 #include <fstream>
 #include <iomanip>
+#include <vector>
 
```

Comment:
Perhaps someone who works with Histomanager can review these changes and whether the new behavior interferes with other uses.
It does not look correct to have the ModuleMapper be a member of the HistoManager, which is a more general code. You could consider deriving a new class (eg. HistoManagerSVTFits : public Histomanager) and make the SVT specific changes there.